tests/PipelineTest.php

```diff
@@ -24,7 +25,8 @@ public function testProcessor(): void
         $rename = new Rename('foo', 'target.field');
         $set = new Set('field4', 324);
 
-        $processors = new Pipeline($this->_getClient());
+        $client = $this->createMock(Client::class);
```

Comment:
Can you share some background on this change?

Reply:
Yes a client mock is sufficient here to satisfy the pipeline object constructor.
No call is made on the elasticsearch server, as only the `toArray()` method is tested here.